components/script/dom/htmltablecellelement.rs

```diff
@@ -77,6 +85,21 @@ impl HTMLTableCellElementMethods for HTMLTableCellElement {
 
     // https://html.spec.whatwg.org/multipage/#dom-tdth-colspan
     make_uint_setter!(SetColSpan, "colspan");
+
+    // https://html.spec.whatwg.org/multipage/#dom-tdth-cellindex
+    fn CellIndex(&self) -> i32 {
+        if let Some(tr) = NodeCast::from_ref(self).GetParentNode() {
+            if let Some(tr) = HTMLTableRowElementCast::to_root(tr) {
+                let this = Some(Root::from_ref(ElementCast::from_ref(self)));
+                for i in 0..tr.Cells().Length() {
```

Comment:
Using `Cells` and `Item` kinda sucks. Why not use `children().position()`? That should be a lot more efficient, and because we compare to `self`, would still be correct.